src/metrics/graphite_metric.ts

```diff
@@ -5,9 +5,6 @@ import * as moment from 'moment';
 
 
 export class GraphiteMetric extends AbstractMetric {
-
-  private _queryParts: string[];
```

Comment:
pleas don`t touch Graphite in this PR, please fix it in another PR. Otherwise you can`t guarantee that you don`t break Graphite here.